package.json

```
@@ -217,6 +217,7 @@
     "test:e2e:dev": "start-server-and-test start http://localhost:3000 cy:open",
     "extract-json-messages": "rimraf dist/locale && cross-env NODE_ENV=extract babel src 'src/**/*.{ts,tsx}' -o /dev/null",
     "transpile-messages": "node scripts/transpile-tx.js",
-    "extract-messages": "npm run extract-json-messages && npm run transpile-messages"
+    "extract-messages": "npm run extract-json-messages && npm run transpile-messages",
+    "plop": "plop"
```

Running plop isn't a goal itself, generating component is ;)

```suggestion
    "generate": "plop"
```